test/ctia/test_helpers/core.clj

```diff
@@ -2,6 +2,7 @@
   (:refer-clojure :exclude [get])
   (:require [ctia.auth :as auth]
             [ctia.auth.allow-all :as aa]
+            [ctia.http.server :as http-server]
             [ctia.properties :as props]
```

Comment:
I think it has to call init!, which is why it is there.  You can see me doing machinations to not have to write properties/properties.  :-P